operator/apis/platform/v1alpha1/central_types.go

```diff
@@ -69,7 +69,7 @@ type Egress struct {
 }
 
 // ConnectivityPolicy is a type for values of spec.egress.connectivityPolicy.
-//+kubebuilder:validation:Enum=Online;Offline
+// +kubebuilder:validation:Enum=Online;Offline
```

Comment:
Does this validation work still?

Reply:
Test are passing so I think it's working